projects/elife.yaml

```diff
@@ -988,21 +988,21 @@ journal-cms:
             rds:
                 type: db.t2.small
                 engine: MySQL
-                version: '5.7'
-                storage: 24 # GB
+                version: '5.7.23'
```

Comment:
`continuumtest` and `prod` instances were stuck on `5.7.22` and `5.7.19` respectively, despite minor version upgrades being turned on. Need a AWS ticket for this to understand what's wrong

Reply:
Ticket has been opened. I've gone through the docs and can't find any reason why it shouldn't have happened or weasel words or exceptions to the rule.

There are AWS forum posts and Stackoverflow posts about this as well with no resolution besides simply doing it yourself.

Reply:
Where's the `:grrr:` emoji when you need it